pyproject.toml

```diff
@@ -30,10 +30,15 @@ dependencies = [
   "prometheus_client==0.20.0",
   "grpcio==1.64.1",
   "grpcio-health-checking==1.62.1",
-  "grpcio-reflection==1.64.1",
+  "grpcio-reflection==1.62.2",
   "transformers",
   "accelerate==0.31.0",
-  "hf-transfer==0.1.6"
+  "hf-transfer==0.1.6",
+  # additional dependencies for OpenTelemetry tracing
+  "opentelemetry-sdk",
+  "opentelemetry-api",
+  "opentelemetry-exporter-otlp",
+  "opentelemetry-semantic-conventions-ai"
```

Comment:
ideally they should be in a separate section?

Reply:
These are not used in this project, are these actually required?

Reply:
Huh, good point. Perhaps they are needed when you start vllm with `--otlp-traces-endpoint` enabled? let me check

Reply:
Yeah look at this: https://github.com/opendatahub-io/vllm/blob/main/vllm/config.py#L1404-L1407